frontend/src/store/index.js

```diff
@@ -123,6 +123,11 @@ export default new Vuex.Store( {
          } );
       },
 
+      updateServiceStatus( state, { appName, serviceName, serviceStatus } ) {
+         const service = state.apps[appName].find(service => service.name == serviceName);
+         Vue.set( service.state, 'status', serviceStatus );
```

Comment:
Should this attribute not be present from the start? What if the service has already be stopped by someone else when loading the UI? But if it _is_ present from the start, `Vue.set` should not be necessary.

Reply:
That's true. I'll fix this.